app/src/Containers/EventSearchContainer.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/native'
+import { ScrollView, SafeAreaView, Button, Text } from 'react-native'
+import TopOfApp from '@/Components/TopOfApp'
+import underDevelopmentAlert from '@/Utils/UnderDevelopmentAlert'
+import DateTimePicker from '@react-native-community/datetimepicker'
+import FreeSearchBar from '@/Components/FreeSearchBar';
```

Comment:
Unneeded semi-colon at the end of this line (the coding style for the front-end app is not to use these)

Reply:
Removed